doc/source/tune/_tutorials/overview.rst

```diff
@@ -48,6 +48,12 @@ Take a look at any of the below tutorials to get started with Tune.
    :figure: /images/pytorch_lightning_small.png
    :description: :doc:`Tuning PyTorch Lightning modules <tune-pytorch-lightning>`
 
+.. customgalleryitem::
+   :tooltip: Model selection and serving with Ray Tune and Ray Serve
+   :figure: /images/serve.png
```

Comment:
serve/logo.svg

Reply:
I tried that before but SVGs don't seem to work with `customgalleryitem`:
`OSError: cannot identify image file 'ray/doc/source/serve/logo.svg'`